VideoWeb/VideoWeb/ClientApp/src/app/services/device-detection.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Logger } from 'src/app/services/logging/logger-base';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class DeviceDetectionService {
```

Comment:
there should already be a device detector service (think it's called `DeviceTypeService`) that wraps the device type library. might want to check that out

Reply:
thanks. Have changed the implementation to use that service instead, and removed the new one